src/App.tsx

```diff
@@ -1,73 +1,93 @@
-import React from 'react';
+import { useState, useRef } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import debounce from 'lodash.debounce';
+import DropdownMenu from './components/DropdownMenu';
+import NoMatch from './components/NoMatch';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const initialPerson: Person = {
+    name: '',
+    born: 0,
+    died: 0,
+    slug: '',
+    fatherName: '',
+    motherName: '',
+  };
+
+  const [selectedPerson, setSelectedPerson] = useState<Person>(initialPerson);
+  const { name, born, died } = selectedPerson;
+  const [isDropdownActive, setIsDropdownActive] = useState(false);
+  const [suggestions, setSuggestions] = useState<Person[]>(peopleFromServer);
```

Comment:
Consider changing it like that to improve readability
```suggestion
  const [selectedPerson, setSelectedPerson] = useState<Person>(initialPerson);
  const [isDropdownActive, setIsDropdownActive] = useState(false);
  const [suggestions, setSuggestions] = useState<Person[]>(peopleFromServer);
  
  const { name, born, died } = selectedPerson;
```